llvm/lib/Target/AIE/AIEEliminateDuplicatePHI.cpp

```diff
@@ -120,26 +120,26 @@ class AIEEliminateDuplicatePHI : public MachineFunctionPass {
   bool processMBB(MachineBasicBlock &MBB, MachineRegisterInfo &MRI,
                   MachineIRBuilder &MIB, GISelObserverWrapper &Observer) {
     bool Changed = false;
-    auto LastPHI = MBB.getFirstNonPHI();
```

Comment:
I would normally expect an iterator with this name to point to the last phi, not the the first non-phi.